libbeat/docs/shared-template-load.asciidoc

```diff
@@ -47,27 +53,42 @@ setup.template.fields: "path/to/fields.yml"
 setup.template.overwrite: false
 ----------------------------------------------------------------------
 
-By default, if a template already exists in the index, it is not overwritten. To overwrite an existing
-template, set `setup.template.overwrite: true` in the configuration file.
+By default, if a template already exists in the index, it is not overwritten. To
+overwrite an existing template, set `setup.template.overwrite: true` in the
+configuration file.
 
-To disable automatic template loading, set `setup.template.enabled: false`.
+To disable automatic template loading, set `setup.template.enabled: false`. If
+you disable automatic template loading, you need to load the template manually.
 
-The options for auto loading the template are not supported if you are using the
-Logstash output.
+See <<configuration-template>> for the full list of configuration options.
+
+The options for auto loading the template are not supported if Logstash output
+is enabled.
 
 [[load-template-manually]]
 ==== Load the template manually
 
-If you disable automatic template loading, you can run the `setup` command to
-load the template manually.
+To load the template manually, run the <<setup-command,`setup`>> command. A
+connection to Elasticsearch is required. If Logstash output is enabled, you need
+to temporarily disable the Logstash output and enable Elasticsearch by using the
+`-E` option. 
+
+The examples here assume that Logstash output is enabled. You can omit the `-E`
+flags if Elasticsearch output is already enabled.
+
+If you are connecting to a secured Elasticsearch cluster, also see
+<<passing-credentials-template-loading>>.
+
+If the host running {beatname_uc} does not have direct connectivity to
+Elasticsearch, see <<load-template-manually-alternate>>.
 
 ifdef::allplatforms[]
 
 *deb, rpm, and mac:*
 
 ["source","sh",subs="attributes"]
 ----
-./{beatname_lc} setup --template
+./{beatname_lc} setup --template -E output.logstash.enabled=false -E 'output.elasticsearch.hosts=["localhost:9200"]'
```

Comment:
decided to show this in the example since it sounds like the most common use case for manual loading is when Logstash output is configured.